test/provider_devdax_memory.cpp

```diff
@@ -349,3 +351,15 @@ TEST_F(test, create_wrong_size_0) {
     EXPECT_EQ(ret, UMF_RESULT_ERROR_INVALID_ARGUMENT);
     EXPECT_EQ(hProvider, nullptr);
 }
+
+HostMemoryAccessor hostAccessor;
+
+static std::vector<ipcTestParams> ipcProxyPoolTestParamsList = {
+    {umfProxyPoolOps(), nullptr, umfDevDaxMemoryProviderOps(),
```

Comment:
Should we also test with other pool implementations, e.g. scalable_pool, jemalloc_pool.

Reply:
Done

Reply:
We should enable them here as I mentioned in this comment: https://github.com/oneapi-src/unified-memory-framework/pull/845/files#r1827620658

Reply:
Done, is it OK now?